# ON MANA, in mm, x/y/z, MTI1 on mana-base, MTI2 on platine
#################
#cam_left /platine :   28     193    14.5
#cam_right/platine :   28    -193    14.5
#MTI2     /platine :   21.2    75     9.5
#platine  /MTI1    :  340      82    59
#MTI1     /origin  :    0       0   637
#GPS      /origin  : -410     140  1140
#################

## constvel
CAMERA_POSE_CONSTVEL: '[6](0, 0, 0, -90, 0, -90)'
CAMERA2_POSE_CONSTVEL: '[6](0, -0.386, 0, -90, 0, -90)'

## mana, mti on platine, platine front or side
#CAMERA_POSE_INERTIAL: '[6](0.0068, 0.118, 0.005, -90, 0, -90)'
#CAMERA2_POSE_INERTIAL: '[6](0.0068, -0.268, 0.005, -90, 0, -90)'

## mti on mana-base, platine front
CAMERA_POSE_INERTIAL: '[6](0.292, 0.193, 0.108, -90, 0, -90)'
CAMERA2_POSE_INERTIAL: '[6](0.292, -0.193, 0.108, -90, 0, -90)'

## mana, mti on mana-base, platine side
#CAMERA_POSE_INERTIAL: '[6](?, ?, 0.108, -90, 0, 0)'
#CAMERA2_POSE_INERTIAL: '[6](?, ?, 0.108, -90, 0, 0)'


## mana, mti on platine, platine front
#GPS_POSE: '[6](-0.6925, 0.06, 0.419, 0, 0, 0)'
## mana, mti on platine, platine side
#GPS_POSE: '[6](?, ?, ?, 0, 0, 0)'
## mana, mti on mana-base
GPS_POSE: '[6](-0.41, 0.14, 0.493, 0, 0, 0)'


## mana, mti on mana-base
ROBOT_POSE: '[6](0,0,-0.647, 0,0,0)'
## mana, mti on platine, platine front
#ROBOT_POSE: '[6](-0.2825, -0.08, -0.74, 0,0,0)'
## mana, mti on platine, platine side
#ROBOT_POSE: '[6](?, ?, -0.74, 0,0,-90)'


## mana, camera left (marlin)
CAMERA1_TYPE: 1
CAMERA1_DEVICE: '0x000a47010b085101'
CAMERA_IMG_WIDTH: 640
CAMERA_IMG_HEIGHT: 480
# (marlin dc1394:0x000a47010b085101, 02/07/2012)
#CAMERA1_INTRINSIC: [4](+3.2528680443177677e+02, +2.4689431763530897e+02, +3.7480458209316453e+02, +3.7469163204199094e+02)
#CAMERA1_DISTORTION: '[3](-2.6401705361046302e-01, +1.2594370560436677e-01, -3.2952194337701121e-02)'
# 19/10/2012
CAMERA1_INTRINSIC: '[4](+3.2703920036560964e+02, +2.4714638294268735e+02, +3.7290284087776149e+02, +3.7271088432359716e+02)'
CAMERA1_DISTORTION: '[3](-2.6188796889984112e-01, +1.2158236902559119e-01, -3.0273706315773623e-02)'
CAMERA1_CALIB: 'calib.data'

CAM_FX: +3.7290284087776149e+02
CAM_FY: +3.7271088432359716e+02
CAM_CX: +3.2703920036560964e+02 
CAM_CY: +2.4714638294268735e+02

CAM_D0: -2.6188796889984112e-01
CAM_D1: +1.2158236902559119e-01
CAM_D2: -3.0273706315773623e-02


## mana, camera right (marlin)
CAMERA2_TYPE: 1
CAMERA2_DEVICE: '0x000a47010b08b047'
CAMERA2_IMG_WIDTH: 640
CAMERA2_IMG_HEIGHT: 480
# mana, camera right (marlin 0x000a47010b08b047, 02/07/2012)
CAMERA2_INTRINSIC: '[4](+3.2218730925843948e+02, +2.4413816511504055e+02, +3.7308305962780344e+02, +3.7295667570897302e+02)'
CAMERA2_DISTORTION: '[3](-2.6603798974663245e-01, +1.2811242206279058e-01, -3.3446717410930311e-02)'
CAMERA2_CALIB: calib.data

## Flea2
#CAMERA1_DEVICE: 0x00b09d010063565b

# SIMU SENSOR
CAMERA1_IMG_WIDTH_SIMU: 640
CAMERA1_IMG_HEIGHT_SIMU: 480
CAMERA1_INTRINSIC_SIMU: '[4](320.0, 240.0, 500.0, 500.0)'
CAMERA1_DISTORTION_SIMU: '[3](-0.25,   0.10, 0.0)'

# initial for all
UNCERT_HEADING: 0.1
UNCERT_ATTITUDE: 0.05

# CONSTANT VELOCITY
UNCERT_VLIN: 0.
UNCERT_VANG: 0.
PERT_VLIN: 2.0
PERT_VANG: 2.0

# INERTIAL (also using UNCERT_VLIN)
#MTI_DEVICE: /dev/serial/by-id/usb-FTDI_USB_Serial_Converter_FTF1IDPL-if00-port0
MTI_DEVICE: /dev/serial/by-id/usb-Keyspan__a_division_of_InnoSys_Inc._Keyspan_USA-19H-if00-port0
ACCELERO_FULLSCALE: 17.0
ACCELERO_NOISE: 0.0109545
GYRO_FULLSCALE: 5.23599
GYRO_NOISE: 0.00772691

UNCERT_GRAVITY: 0.01
UNCERT_ABIAS: 0.01
UNCERT_WBIAS: 0.01
PERT_AERR: 1.0
PERT_WERR: 1.0
PERT_RANWALKACC: 0.0
PERT_RANWALKGYRO: 0.0

# mana
IMU_TIMESTAMP_CORRECTION: 0.00
GPS_TIMESTAMP_CORRECTION: 0.0
POS_TIMESTAMP_CORRECTION: -0.1

ODO_CALIB: '[4](1.0, 1.0, 1.0, 1.0)'


# SIMU INERTIAL
SIMU_IMU_TIMESTAMP_CORRECTION: 0.0
SIMU_IMU_FREQ: 120.0
SIMU_IMU_GRAVITY: 9.81
SIMU_IMU_GYR_BIAS: 0.0
SIMU_IMU_GYR_BIAS_NOISESTD: 0.0
SIMU_IMU_GYR_GAIN: 0.0
SIMU_IMU_GYR_GAIN_NOISESTD: 0.0
SIMU_IMU_RANDWALKGYR_FACTOR: 1.0
SIMU_IMU_ACC_BIAS: 0.0
SIMU_IMU_ACC_BIAS_NOISESTD: 0.0
SIMU_IMU_ACC_GAIN: 0.0
SIMU_IMU_ACC_GAIN_NOISESTD: 0.0
SIMU_IMU_RANDWALKACC_FACTOR: 1.0
